compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -380,30 +381,32 @@ internal class RootNodeOwner(
 
             @OptIn(InternalTextApi::class)
             override suspend fun startInputMethod(request: PlatformTextInputMethodRequest): Nothing {
-                innerSessionMutex.withSessionCancellingPrevious(
+                innerSessionMutex.withSessionCancellingPrevious<Nothing>(
                     sessionInitializer = { null }
                 ) {
-                    // Currently TextInputService is used for keyboard show/hide actions and for
-                    // backward compatibility by the LocalTextInputService.
-                    // startInput and stopInput calls are required to properly configure the service
-                    // and allow it to pass keyboard show/hide calls to the PlatformTextInputService.
-                    textInputService.startInput()
-                    launch(start = CoroutineStart.UNDISPATCHED) {
-                        suspendCancellableCoroutine<Nothing> {
-                            it.invokeOnCancellation {
-                                textInputService.stopInput()
+                    coroutineScope {
+                        // Currently TextInputService is used for keyboard show/hide actions and for
+                        // backward compatibility by the LocalTextInputService.
+                        // startInput and stopInput calls are required to properly configure the service
+                        // and allow it to pass keyboard show/hide calls to the PlatformTextInputService.
+                        launch(start = CoroutineStart.UNDISPATCHED) {
+                            suspendCancellableCoroutine<Nothing> {
+                                textInputService.startInput()
+                                it.invokeOnCancellation {
+                                    textInputService.stopInput()
+                                }
```

Comment:
@ASalavei Please note the changes here. Hopefully I didn't break anything. The changes are because IDEA was complaining about the implicit `Nothing` and ambiguous coroutine context.